src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -1,64 +1,74 @@
 // Libraries
-import React, {FC, useEffect} from 'react'
-import {useSelector, useDispatch} from 'react-redux'
+import React, {FC, useEffect, useState} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
 
 // Components
 import {
-  FlexBox,
   AlignItems,
-  FlexDirection,
   ComponentSize,
+  FlexBox,
+  FlexDirection,
   JustifyContent,
 } from '@influxdata/clockface'
-import UsersProvider from 'src/users/context/users'
 import LabeledData from 'src/organizations/components/OrgProfileTab/LabeledData'
 import CopyableLabeledData from 'src/organizations/components/OrgProfileTab/CopyableLabeledData'
-import DeletePanel from 'src/organizations/components/OrgProfileTab/DeletePanel'
+import {DeletePanel} from 'src/organizations/components/OrgProfileTab/DeletePanel'
 
-// Types
-import {RemoteDataState} from 'src/types'
+// Notifications
+import {orgDetailsFetchError} from 'src/shared/copy/notifications'
+import {notify} from 'src/shared/actions/notifications'
 
-// Utils
-import {CLOUD} from 'src/shared/constants'
+// Providers
+import UsersProvider from 'src/users/context/users'
 
 // Selectors
 import {getMe} from 'src/me/selectors'
 import {getOrg} from 'src/organizations/selectors'
-import {
-  selectCurrentIdentity,
-  selectQuartzOrgDetailsStatus,
-} from 'src/identity/selectors'
+import {selectCurrentOrg} from 'src/identity/selectors'
 
 // Thunks
 import {getCurrentOrgDetailsThunk} from 'src/identity/actions/thunks'
 
+// Types
+import {RemoteDataState} from 'src/types'
+
+// Constants
+import {CLOUD} from 'src/shared/constants'
+
 // Styles
 import 'src/organizations/components/OrgProfileTab/style.scss'
 
 const OrgProfileTab: FC = () => {
   const me = useSelector(getMe)
   const org = useSelector(getOrg)
-  const {org: quartzOrg} = useSelector(selectCurrentIdentity)
-  const orgDetailsStatus = useSelector(selectQuartzOrgDetailsStatus)
-
+  const quartzOrg = useSelector(selectCurrentOrg)
   const dispatch = useDispatch()
 
-  const identityOrgId = quartzOrg.id
+  // Data about the user's organization is intentionally re-fetched when this component mounts again.
+  const [orgDetailsStatus, setOrgDetailsStatus] = useState(
+    RemoteDataState.NotStarted
+  )
```

Comment:
We now use local state to determine whether we make the API call to get the user's org details, not redux.

The data itself still lives in redux. In theory we could tear down portions of the reducer and put a few of the `org` properties in local state in this component now. However, that's confusing (because half of the org properties will live in redux, and half won't), and wasteful because there will likely be other pages where we need to use this data later.